### DISCUSSION TITLE
开源之道（2016.5.27) - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#103] [Announcements]
# 开源之道（2016.5.27) - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_5/2016_5_27/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

开始主动应用称赞的力量,
先发现所有信息源中值得称赞的点...

其它的, 先忽略.
